src/ocean/net/server/unix/UnixConnectionHandler.d

```diff
@@ -62,13 +62,40 @@ import ocean.core.array.Mutation : copy;
 /// Provides basic command handling functionality for unix socket commands.
 public class BasicCommandHandler
 {
-    /// Alias for a command handler delegate.
+    /// Alias for an interactive command handler delegate.
     public alias void delegate ( cstring,  void delegate (cstring),
-            void delegate (ref mstring)) Handler;
+            void delegate (ref mstring)) InteractiveHandler;
 
-    /// Map of command name to handler response delegate.
+    /// Alias for a non-interactive command handler delegate.
+    public alias void delegate ( cstring, void delegate (cstring) ) Handler;
```

Comment:
You change definition of a public alias, how this is not a breaking change?

Reply:
Because it changes it back to what it was in the v3.x.x.

Reply:
https://github.com/sociomantic-tsunami/ocean/blob/6a5371e43c1e28d8d99a676c67566306ae791e7e/src/ocean/net/server/unix/UnixConnectionHandler.d#L65

Reply:
It's v4.x.x, not released yet. It's changing it back to the definition we have in v3.x.x

Reply:
Ah, I see. Would it be less confusing to fixup it into original commit upon next rebase?

Reply:
Yeah, perhaps that's what we could do. I'll update the commit title.

Reply:
Updated.